lib/ci-helper.ts

```diff
@@ -317,7 +317,7 @@ export class CIHelper {
 
         // Identify branch in maintainer repo
         const maintainerBranch = `refs/remotes/${this.config.repo.maintainerBranch}/`;
-        const maintainerRepo = `${this.config.repo.maintainerBranch}.${this.config.repo.name}`;
+        const maintainerRepo = `${this.config.repo.owner}/${this.config.repo.name}`;
```

Comment:
Unfortunately, now line 329 is not correct.  Do you want to fix that?

Reply:
You mean because it mentions `gitgitgadget/git` to contain two separate branches pointing to the same commit?

I would think the comment now to be "more correct": We did have instances in the past where the Azure Pipeline mirrored a branch with a typo in its name from `gitster/git` to `gitgitgadget/git`, later the typo was fixed and the branch with the fixed name was mirrored, too, but while in `gitster/git` the branch with the incorrect name was deleted, it remained in the `gitgitgadget/git` fork. In such an instance, the comment should not claim that there are two candidate branches in `gitster/git` when they really exist only in `gitgitgadget/git`.

Reply:
> Unfortunately, now line 329 is not correct.

Okay, maybe not.  I went back to PR #887 where I introduced this bug and see that that line had a slash as well.  It seems there were a couple of other places (`misc-helper.ts`) where `git.git` was used and I probably copied from there.  Sorry my multiple reviews of the changes missed this.

Reply:
No apologies necessary, we all work together to make GitGitGadget better, right?